layouts/_default/baseof.css

```diff
@@ -119,6 +119,13 @@ html {
   scroll-behavior: smooth;
 }
 
+:target:before {
```

Comment:
`before` is technically a pseudo *element*, so the correct syntax here is `:target::before`. See https://devdocs.io/css/pseudo-elements